app/common/elasticsearchclient.py

```diff
@@ -398,8 +398,11 @@ def autocomplete(self,
             data = res['suggest'][0]['options']
         return SimpleResult(None, params, data)
 
-    def get_gene_info(self, gene_ids, facets=False, **kwargs):
+    def get_gene_info(self, gene_ids, **kwargs):
         params = SearchParams(**kwargs)
+
+        facets = kwargs.get('facets') or False
```

Comment:
Actually I cannot see anymore any use of the facets here, so they could just be removed, together with the `self._get_gene_info_agg()` function, that is actually broken.
Need to remove lines 412-413 as well.